src/audio/module_adapter/module/generic.c

```diff
@@ -495,20 +495,52 @@ EXPORT_SYMBOL(module_set_configuration);
 
 int module_bind(struct processing_module *mod, struct bind_info *bind_data)
 {
+	int ret;
 	const struct module_interface *const ops = mod->dev->drv->adapter_ops;
 
+	switch (bind_data->bind_type)
+	{
```

Comment:
style: please move the brace to the `switch` line and align `case` lines with `switch` - see examples around the code base. Same below